docs/AnaplanExport-batchsource.md

```diff
@@ -0,0 +1,139 @@
+# Anaplan source
+
+Description
+-----------
+Source plugin for loading data from Anaplan model into CDF pipeline.
+
+This plugin is built upon the existing [GCS source plugin](https://github.com/data-integrations/google-cloud/tree/develop/src/main/java/io/cdap/plugin/gcp/gcs/source).
+
+The data exported from Anaplan model will be firstly materialized in the given GCS bucket, 
+and the inbound data stream to the CDF pipeline will be sourced from the file.
+
+GCP Credentials
```

Comment:
Should we be more clear about how the service account should work via auto-detect for dataproc?  Or is this a recommendation only?  Or do we recommend people to use the json key file?

Reply:
I got the GCP Credential and GCS & General Properties session directly from the doc for existing GCS source plugin.

As I know the auto-detect is the default setting and the credential will be fetched in the runtime. The credential to be fetched is from the GCP env that runs this CDF instance. Added this detail.